packages/sku/lib/program/commands/pre-commit/pre-commit.action.js

```diff
@@ -0,0 +1,13 @@
+const preCommit = require('../../../preCommit');
+const { configureProject } = require('../../../utils/config-validators');
+
+const preCommitAction = async () => {
+  await configureProject();
+  try {
+    await preCommit();
+  } catch (e) {
+    process.exit(1);
```

Comment:
Is it possible for a thrown error to not otherwise have been logged out? It feels like this could end up hiding errors, where the command fails with no reason why.